setup.cfg

```diff
@@ -29,6 +29,8 @@ packages = find:
 package_dir = = src
 include_package_data = true
 python_requires = >= 3.6
+install_requires =
+    itsdangerous ~= 2.0.0
```

Comment:
I think this also rejects 2.1 even though that's not expected to contain breaking changes. I'd use `~= 2.0` to allow anything `<3`...

Reply:
Yes, you are right: https://www.python.org/dev/peps/pep-0440/#compatible-release